### ISSUE TITLE
Version conflict on newest version (deface compatibility)

### ISSUE THREAD

**Issue by fwolfst** [solidusio-contrib/solidus_static_content#65]
Installing latest solidus and the solius_static_content extension leads to

```
Bundler could not find compatible versions for gem "deface":
  In snapshot (Gemfile.lock):
    deface (= 1.5.3)

  In Gemfile:
    solidus_auth_devise was resolved to 2.4.0, which depends on
      deface (~> 1.0)

    solidus_print_invoice was resolved to 1.0.2, which depends on
      deface

    solidus_static_content was resolved to 1.0.0, which depends on
      deface (~> 1.0.2)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

I believe the "culprit" here is the dependency on deface `~> 1.0.2`, where hopefully `~> 1.0` would suffice.

**Comment by memotoro** [comment]
Hello @fwolfst 

What version of the solidus_static_content are you pulling? 

My gem is pointing to master branch with the commit the latest commit and it is working for me. Could you have a look to your lock file and check if that works? Regards

```
GIT
  remote: https://github.com/solidusio-contrib/solidus_static_content.git
  revision: 4ffe31c3b04b7ddbee36931689eb2be5ef5900b8
  branch: master
  specs:
    solidus_static_content (1.0.0)
      deface (~> 1.0)
      solidus_core (>= 2.0.0, < 3)
      solidus_support (~> 0.5)
```

**Comment by fwolfst** [comment]
I just required it as per README, but will try with master branch.
`solidus_static_content` never made it into Gemfile.lock (I think this only happens on successfull dependency resolution and installation).
I then manually installed it (`gem install solidus_static_content`), which will result in:
```
gem info solidus_static_content

*** LOCAL GEMS ***

solidus_static_content (1.0.0)
    Authors: Peter Berkenbosch, Roman Smirnov
    Homepage: http://spreecommerce.com/extensions/139-static-content
    License: BSD-3
    Installed at: /home/XXXXX/gems/ruby-2.7.1

    Extention to manage the static pages for your Spree shop.
```

**Comment by fwolfst** [comment]
Okay, `gem 'solidus_static_content', github: 'solidusio-contrib/solidus_static_content'` in `Gemfile` does the trick, but this means a new version should be released or the README updated
.

**Comment by memotoro** [comment]
Hi @fwolfst 

I think if you install it as the README you will get the latest release that is `1.0.0` as it is the only release available in the release section. You could go in more detail in your `Gemfile` to point to a particular branch, tag or version like this.
```
gem 'solidus_static_content', github: 'solidusio-contrib/solidus_static_content', branch: 'master'
```
Change your `Gemfile` and then run the following command:
```
bundle update
``` 
Be aware that the previous command will attempt to update other gems as well, just update and commit your changes for the gem you want. Just make sure your lock file is not completely broken after the update.

Regards

**Comment by memotoro** [comment]
Hi @fwolfst 

I guess there was not release with the latest features and bug fixes, so I guess master is the latest.

I'm glad is working for you now. 

Regards

**Comment by fwolfst** [comment]
Thanks @memotoro . It always gives me a bit of creeps if I depend on the github `master` branch of a gem. Is there anything that can be done for a "proper" 1.0.1 / 1.1 release?

**Comment by memotoro** [comment]
Hi @fwolfst I'm not sure how is the release process, I guess @aldesantis or @kennyadsl could help here? I always bug them with thing like this. I'm not sure how the team decides to create a release or how this fit in the release roadmap they have. Regards

**Comment by aldesantis** [comment]
@fwolfst @memotoro hey there, we are slowly releasing new versions of all extensions. I'll give this a go tomorrow if that works!

**Comment by fwolfst** [comment]
> I guess there was not release with the latest features and bug fixes, so I guess master is the latest.
> 
> I'm glad is working for you now. I guess you could close this bug as it is working now?
I'd wait till release.

Thanks so far, that was quick and concise help and response.

**Comment by rsmithlal** [comment]
I ran into the same issue, and running bundle update instead of bundle worked to resolve the dependency issue for me.

**Comment by aldesantis** [comment]
I have just released 2.0.0 with all the latest improvements. It's a major because I also took the chance to remove all deprecated modules and code, so a few things may break here and there, but changing the require paths and namespaces should be enough to fix it.
